code/modules/surgery/generic.dm

```diff
@@ -61,8 +61,8 @@
 	allowed_tools = list(
 		/obj/item/weapon/scalpel/manager = 100
 	)
-	min_duration = 80
-	max_duration = 120
+	min_duration = 15
```

Comment:
This feels waaaay too fast even if it's the incision management system. Make it a bit slower, maybe 30-70?